.tests/cypress/integration/header/cart.spec.js

```diff
@@ -0,0 +1,12 @@
+describe('HEADER - Cart header link', () => {
+
+    it('clicks on cart button.', () => {
+      cy.visit('/')
+      cy.get('[data-testid="minicart-link"]').as('cartButton')
```

Comment:
Wrong data standards.

Reply:
Fixed. Also removed wait time and used `cy.route` instead.